tests/readBison.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+readBison() test
+--SKIPIF--
+<?php if (!extension_loaded("parle")) print "skip"; ?>
+--FILE--
+<?php 
+use Parle\Parser;
+use Parle\Lexer;
+
+$p = new Parser;
+
+$p->readBison("%%\n;start: 'a';%%\n");
+$p->build();
+
+$lex = new Lexer;
+$lex->push("a", $p->tokenId("'a'"));
+$lex->build();
+
+echo $p->validate("a", $lex) . "\n";
```

Comment:
Could you switch to using `var_dump` instead? It'll show the type of the returned value as well.

Thanks

Reply:
The test now does a dump() instead and the test also supports UTF-32